oidc-controller/api/verificationConfigs/router.py

```diff
@@ -1,69 +1,65 @@
-from fastapi import APIRouter, Depends
+from fastapi import APIRouter, HTTPException
 from fastapi import status as http_status
-
 from ..core.models import StatusMessage
 
 from .crud import VerificationConfigCRUD
-from .dependencies import get_verification_configs_crud
 from .models import (
-    VerificationConfigCreate,
     VerificationConfigPatch,
     VerificationConfigRead,
+    VerificationConfig,
 )
 
 router = APIRouter()
 
 
 @router.post(
-    "", response_model=VerificationConfigRead, status_code=http_status.HTTP_201_CREATED
+    "/",
```

Comment:
Status code seems to be missing, is this on purpose?

Reply:
fixed, good catch!